test/allowlist_for_publicAPI.json

```diff
@@ -353,6 +353,7 @@
     "IntType",
     "List",
     "ListType",
+    "Sequence",
```

Comment:
You should not be adding anything to this file in general.

Reply:
Actually, I'm not sure why to add this to public API. However, to follow the existing pattern I did this.

Should I just hide it from `__all__`?